platforms/Windows/sdk-arm64.wxs

```diff
@@ -211,6 +211,9 @@
         <File Id="time.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\time.h" Checksum="yes" />
       </Component>
 
+      <Component Id="generic_base.h" Directory="WindowsSDK_usr_include_os" Guid="a136a05a-acc4-489b-96fc-1413f8421715">
+        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\os\generic_base.h" Checksum="yes" />
```

Comment:
`dispatch` component is not needed here, as it's a peer to `generic_unix_base.h` and `generic_win_base.h`, which are both in `usr\lib\swift\os`, not `usr\lib\swift\dispatch\os`
```suggestion
        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\os\generic_base.h" Checksum="yes" />
```